### DISCUSSION TITLE
Saving to/hydrating from the browser cache

### DISCUSSION THREAD

**Discussion by okbytes** [gilbarbara/react-inlinesvg#177] [Ideas]
I didn't realize when first implementing this package that the _cache_ is in memory. Thus, all the icons used in our app need to be freshly downloaded when it's a new session or a page refresh. We're a single page app, so this happens quite a lot.

I'm not sure how the browser cache works and what something like this would look like. Any tips/experiences welcome. :-)

Thoughts?

**Comment by gilbarbara** [comment]
Hey @okbytes 

Multiple instances cache and browser permanent cache added in v4.x
